docs/reference/aggregations/metrics/boxplot-aggregation.asciidoc

```diff
@@ -0,0 +1,204 @@
+[role="xpack"]
+[testenv="basic"]
+[[search-aggregations-metrics-boxplot-aggregation]]
+=== Boxplot Aggregation
+
+A `boxplot` metrics aggregation that computes boxplot of numeric values extracted from the aggregated documents.
+These values can be extracted either from specific numeric fields in the documents, or be generated by a provided script.
+
+The `boxplot` aggregation returns essential information for making a https://en.wikipedia.org/wiki/Box_plot[box plot]: minimum, maximum
+median, first quartile (25th percentile)  and third quartile (75th percentile) values.
+
+==== Syntax
+
+A `boxplot` aggregation looks like this in isolation:
+
+[source,js]
+--------------------------------------------------
+{
+    "boxplot": {
```

Comment:
I wonder, should we snake_case this to `box_plot`?  I personally don't really like snake casing but it would be more consistent (`date_histogram`, `significant_terms`, etc).

Reply:
Wikipedia states both spellings and `boxplot` was shorter, but I don't have strong feelings about it.

Reply:
In that case, boxplot it is.  I like shorter too